Dockerfile

```
@@ -7,7 +7,7 @@ FROM golang:1.19 as builder
 COPY . /go/src/github.com/registrobr/rdap-client
 WORKDIR /go/src/github.com/registrobr/rdap-client
 RUN mkdir /apps
-RUN go build -mod=vendor -ldflags="-w -s" -o /apps/rdap-client
+RUN go mod vendor && go build -mod=vendor -ldflags="-w -s" -o /apps/rdap-client
```

We could also drop the `-mod=vendor` since [it was removed from the repo](https://github.com/registrobr/rdap-client/pull/15).

```suggestion
RUN go build -ldflags="-w -s" -o /apps/rdap-client
```